app/src/main/java/com/cdhiraj40/leetdroid/ui/fragments/user/ContestDetailsFragment.kt

```diff
@@ -27,8 +27,8 @@ class ContestDetailsFragment : Fragment(), ContestHistoryAdapter.OnItemClicked {
     private lateinit var fragmentContestDetailsBinding: FragmentContestDetailsBinding
     private lateinit var contestRankingJson: ContestRankingModel
     private lateinit var contestHistoryAdapter: ContestHistoryAdapter
-    private lateinit var generalErrorView: View
```

Comment:
Why are you removing the error view? It should exist and be visible whenever an exception occurs. It should not be removed.

Reply:
Ok